src/modules/commander/commander.cpp

```diff
@@ -1158,7 +1158,7 @@ int commander_thread_main(int argc, char *argv[])
 #endif
 
 	if (argc > 2) {
-		if (!strcmp(argv[3],"-hil")) {
+		if (!strcmp(argv[2],"-hil")) {
```

Comment:
Was it checked on Pixhawk yet? Or should we #ifdef this for QuRT? Yes the removed line did segfault.